dotcom-rendering/cypress.config.js

```diff
@@ -34,25 +37,16 @@ module.exports = defineConfig({
 
 			const webpackConfig = webpackPreprocessor.defaultOptions;
 			webpackConfig.webpackOptions.resolve = {
-				extensions: ['.ts', '.js'],
+				extensions: ['.js', '.ts', '.tsx', '.cts', '.ctsx'],
 			};
-			const rules = webpackConfig.webpackOptions.module.rules;
-			rules[0].exclude = babelExclude;
-
-			// Adding this here so that we can import the fixture in the sign-in-gate.cy.js file
-			rules.push({
-				test: path.resolve(
-					__dirname,
-					`./fixtures/generated/articles/Standard.ts`,
-				),
-				exclude: ['/node_modules/'],
-				loader: 'ts-loader',
-				options: {
-					compilerOptions: {
-						noEmit: false,
-					},
+			webpackConfig.webpackOptions.module.rules = [
+				{
+					test: /\.c?tsx?$/,
+					exclude: babelExclude,
+					use: swcLoader,
 				},
-			});
+			];
+
```

Comment:
What's happening here? Is there a change from `ts-loader` to `swcLoader` and is that related to the ESM changes?